src/components/navBar/SearchBar.jsx

```diff
@@ -0,0 +1,60 @@
+import React, { useContext, useState } from "react";
+import youtubeApi from "../../utils/Youtube";
+import VideoContext from "../../providers/VideoContext";
+import "../styles/SearchBar.css";
+const SearchBar = () => {
+  const [inputword, setInputWord] = useState("");
+  const { setVideoMetaInfo } = useContext(VideoContext);
+  const { changinggSetChangingg } = useContext(VideoContext);
+  const { setFavoritesFlag } = useContext(VideoContext);
+
+  const onSearch = async (e) => {
+    console.log("do request with params", inputword);
+
+    const params = {
+      q: inputword
+    };
+
+    const varResponse = await youtubeApi.get("/search", {
```

Comment:
I would remove the var prefix for this const. It makes a confusing name for a variable or constant.